src/components/AdminLayout/index.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { Box } from '@mui/system';
+import HomeIcon from '@mui/icons-material/Home';
+import PersonIcon from '@mui/icons-material/Person';
+import PeopleIcon from '@mui/icons-material/People';
+import AdminIcon from '@mui/icons-material/AdminPanelSettings';
+import { SideMenu, SideMenuLabels } from 'components/SideMenu/SideMenu';
+import { admin, home, profile, users } from "routes/Roots";
+import styles from './style.module.css';
+
+interface CustomLayoutProps {
+  children: React.ReactNode
+}
+
+const MenuLabels: SideMenuLabels[] = [
+  { label: "HOME", path: home(), icon: HomeIcon },
+  { label: "USERS", path: users(), icon: PeopleIcon},
+  { label: "PROFILE", path: profile(), icon: PersonIcon },
+  { label: "ADMIN", path: admin(), icon: AdminIcon },
+]
+
+const CustomLayout: React.FC<CustomLayoutProps> = ({children}) => {
```

Comment:
For consistency and kinda a standard that Rodrigo wants to maintain, all functional components should be written as follows
`function MyComponent ({propA, propB}: MyComponentProps) {...}`

Do this for all the components created for this PR

Reply:
No problem! on it 😅